Assets/Resources/Models/ScifIPanel/obj.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
last time my Unity deleted this file, because obj file didn't exist
are you sure that this .meta is necessary?

Reply:
No idea, it was automatically added when i pulled the latest version.